Duplicati/Server/Program.cs

```diff
@@ -302,10 +303,54 @@ public static int RealMain(string[] _args)
             }
 
             if (UpdatePoller != null && UpdatePoller.IsUpdateRequested)
+            {
+                if (Platform.IsClientWindows && WindowsServiceExists("Duplicati"))

```

Comment:
Would it be safer to reference `WindowsService.ServiceControl.SERVICE_NAME` instead of hard-coding `"Duplicati"` here?

https://github.com/duplicati/duplicati/blob/beaf03562fdcf4425e962085bdf7175d6a465f49/Duplicati/WindowsService/ServiceControl.cs#L15

Reply:
Yes, that's a good improvement.  I wasn't aware that was available.

I should also note that .Net has a library called System.ServiceProcess.dll, which has a managed code object called "ServiceController" for working with services.  Even though that code would be better than my solution of invoking a command line program and parsing its output, I didn't know if adding a reference to that new dll would make life more difficult for anyone since it would be the only code in this .Net Project that requires that reference.  If adding that new dll reference would be fine with everyone,  the code could be rewritten to use that instead.

Reply:
If using System.ServiceProcess.dll cleans things up, I would be in favor of referencing it.